src/alfasim_sdk/__init__.py

```diff
@@ -517,4 +520,7 @@ def get_alfasim_sdk_api_path():
     "load_pvt_model_table_parameters_description_from_alfatable",
     "tab",
     "tabs",
+    "CorrelationsOilViscosity",
```

Comment:
This list should be kept in alphabetical order.

Reply:
Also note as lower case values come after capitalized ones.

Reply:
ok